crates/biome_js_formatter/tests/specs/prettier/js/do/call-arguments.js.prettier-snap

```diff
@@ -31,3 +31,7 @@ expect(
     bar;
   },
 ).toThrow(ReferenceError);
+
+useImperativeHandle(ref, () => {
```

Comment:
These tests are coming from prettier repository, so they should not be modified

Reply:
My bad 😅 I thought that since we are dealing with a difference in formatting with prettier, I should place the test there